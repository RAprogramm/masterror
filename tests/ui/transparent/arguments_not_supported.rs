use masterror::Error;

#[derive(Debug, Error)]
#[error(transparent, code = 42)]
struct TransparentWithArgs(#[from] std::io::Error);

fn main() {}
