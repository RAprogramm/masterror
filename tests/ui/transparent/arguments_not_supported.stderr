error: format arguments are not supported with #[error(transparent)]
 --> tests/ui/transparent/arguments_not_supported.rs:4:20
  |
4 | #[error(transparent, code = 42)]
  |                    ^
